spec/requests/api/picture_spec.rb

```diff
@@ -63,26 +67,10 @@ def expect_result_to_include_picture_href(source_id)
   end
 
   describe 'POST /api/pictures' do
-    # one pixel png image encoded in Base64
```

Comment:
Ah, that makes sense. So you'd need a really really really long line :stuck_out_tongue_closed_eyes: 

Or, you could use this pattern:

```ruby
"foo"\
"bar"\
"baz" # => "foobarbaz"
```